src/substrait/textplan/parser/grammar/CMakeLists.txt

```diff
@@ -11,15 +11,15 @@ set(GRAMMAR_DIR ${CMAKE_BINARY_DIR}/src/substrait/textplan/parser/grammar)
 # using /MD flag for antlr4_runtime (for Visual C++ compilers only)
 set(ANTLR4_WITH_STATIC_CRT OFF)
 
-set(ANTLR4_TAG 4.12.0)
+set(ANTLR4_TAG 4.13.0)
 set(ANTLR4_ZIP_REPOSITORY
-    https://github.com/antlr/antlr4/archive/refs/tags/4.12.0.zip)
+    https://github.com/antlr/antlr4/archive/refs/tags/4.13.0.zip)
```

Comment:
Nit: you could use the var set above for the version here.

Reply:
Done.